screen-pop-app/screen-pop/src/grantLogin.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect, useRef } from "react";
+import axios from "axios";
+import "./App.css";
+import { Channel } from "./notification";
+import { ConfigData,  ConversationManager } from "./ConversationManager";
+
+const CLIENT_ID = "81a7fe1e-497b-4b99-aef0-eb67192750e9";
+const ENVIRONMENT = "mypurecloud.com"
+
+function getParameterByName(name: string) {
+  // eslint-disable-next-line
```

Comment:
What linter rule is being disabled here?

Reply:
Unnecessary escape character: \[  no-useless-escape

Reply:
can you fix it instead of ignoring it?

Reply:
alright I'll look into it